src/main/java/org/mockito/internal/junit/UnnecessaryStubbingsReporter.java

```diff
@@ -25,7 +25,7 @@
 
     public void validateUnusedStubs(Class<?> testClass, RunNotifier notifier) {
         Collection<Invocation> unused = new UnusedStubbingsFinder().getUnusedStubbingsByLocation(mocks);
-        if (unused.size() == 0) {
+        if (unused.isEmpty()) {
```

Comment:
Thanks for this change and similar ones related to isEmpty! The code is more readable this way.